tests/mock_crust/churn.rs

```diff
@@ -94,8 +94,8 @@ fn did_receive_get_success(node: &TestNode,
 
 fn target_group(nodes: &[TestNode], target: Authority) -> Vec<&TestNode> {
     nodes.iter()
-         .filter(|n| n.routing_table().is_recipient(&target.to_destination()))
-         .collect_vec()
+        .filter(|n| n.routing_table().is_recipient(&target.to_destination()))
+        .collect_vec()
```

Comment:
I am - it's the new version that does this, the previous version gave me results like the one being overwritten.